concrete/src/Page/Sitemap/Element/SitemapPage.php

```diff
@@ -0,0 +1,307 @@
+<?php
+
+namespace Concrete\Core\Page\Sitemap\Element;
+
+use Concrete\Core\Error\UserMessageException;
+use Concrete\Core\Page\Page;
+use Concrete\Core\Url\UrlInterface;
+use DateTime;
+use SimpleXMLElement;
+
+class SitemapPage extends SitemapElement
+{
+    /**
+     * The page.
+     *
+     * @var \Concrete\Core\Page\Page
+     */
+    protected $page;
+
+    /**
+     * The URL of the page.
+     *
+     * @var \Concrete\Core\Url\UrlInterface
+     */
+    protected $url;
+
+    /**
+     * The last modification date/time.
+     *
+     * @var \DateTime|null
+     */
+    protected $lastMod;
```

Comment:
Personally I don't like abbrevations (why otherwise not 'changeFreq' below). Maybe `lastModifiedAt` or `lastModified` or `modifiedAt`?

Reply:
`lastModifiedAt` or `lastModifiedOn`?

Reply:
It's date + time, so I'd say `at`. (this is also a convention Laravel uses, btw)